### PR TITLE
Release management #0: Preparation of the version file for 1.13.3 release

### PR DESCRIPTION
**Pull request by bari12**
Release management #0: Preparation of the version file for 1.12.4 release

### CODE REVIEW

**Comment by vingar** [comment]
conflict

File: .gitignore [hunk 1]
```diff
@@ -5,26 +5,58 @@
 # You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
 #
 # Authors:
-# - Mario Lassnig (CERN PH-ADP),    <mario.lassnig@cern.ch>,   2011
-# - Vincent Garonne (CERN, PH-ADP), <vincent.garonne@cern.ch>, 2011
+# - Mario Lassnig, <mario.lassnig@cern.ch>, 2012-2015
+# - Vincent Garonne, <vincent.garonne@cern.ch>, 2012
```
Comment (vingar) [inline .gitignore]: from git-log:
Mario Lassnig <mario.lassnig@cern.ch>, 2012-2016
Vincent Garonne <vincent.garonne@cern.ch>, 2012-2014
Martin Barisits <martin.barisits@cern.ch>, 2012-2015
Wen Guan <wguan.icedew@gmail.com>, 2015-2015

**Comment by vingar** [comment]
I didn't realize that we have so much changes in this pull request. Is it expected ?

**Comment by bari12** [comment]
It's the same with @sprenner pull request against next #90. I am not sure what is happening, as this was not displayed like this last week; I wonder if github changed something. I will check.

**Comment by bari12** [comment]
Ok, I rebased against the actual upstream/next, this works then. I think our origin/master branches are a bit wonky, I will change the create-patch/next-branch script to always make a full reset of the branch against the upstream one.

**Review by vingar — APPROVED** [review]

**Pull request merged**